idd/Energy+.idd.in

```diff
@@ -15893,7 +15898,6 @@ SurfaceProperty:HeatTransferAlgorithm:SurfaceList,
        \memo EMPD (Effective Moisture Penetration Depth with Conduction Transfer Functions).
        \memo Advanced/Research Usage: CondFD (Conduction Finite Difference)
        \memo Advanced/Research Usage: HAMT (Combined Heat And Moisture Finite Element)
-       \min-fields 3
```

Comment:
Why remove min-fields here?  This will change setting of defaults by the input processor.  Do all of the defaults gets initialized in source code?